react/features/base/session/actionTypes.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * FIXME.
+ *
+ * {
+ *     type: SET_SESSION,
+ *     session: {
+ *         url: {string},
+ *         state: {string},
+ *         ...data
+ *     }
+ * }
+ * @public
+ */
+export const SET_SESSION = Symbol('SET_SESSION');
```

Comment:
Having this single action makes things a bit weird in the eye IMHO. How about CREATE_SESSION / UPDATE_SESSION / DESTROY_SESSION ?

Reply:
I'm open to that. It's just that it seems a bit less code, because CREATE_SESSION equals SESSION_WILL_START and DESTROY_SESSION equals SESSION_ENDED/FAILED state.